lib/Runtime/Executor/ThreadPoolExecutor.cpp

```diff
@@ -179,8 +202,6 @@ void ExecutionState::insertIntoResultCtx(Placeholder *P, Tensor &&T) {
 
   if (tensor) {
     *tensor = std::move(T);
-  } else {
-    resultCtx_->getPlaceholderBindings()->insert(P, std::move(T));
```

Comment:
I deleted this so that input bindings from terminal DAG nodes don't get propagated to the final result context.